.mk/code_analysis.mk

```diff
@@ -4,11 +4,11 @@ lint-fix:
 
 .PHONY: lint-install
```

Comment:
Can these go in a separate PR please.

Reply:
Yep, Let the build finish and I'll update this PR.

Reply:
done